howto/install-appliance/azure.md

```diff
@@ -78,14 +78,21 @@ Azure will prompt you to download your private key before it starts creating the
 
 When deployment is complete, you can log on to the machine and install the Anbox Cloud Appliance.
 
-## Log on to the machine
+## Connect to the virtual machine using SSH
 
-To install the Anbox Cloud Appliance, you must log on to the virtual machine that you just created.
+To install the Anbox Cloud Appliance, you must connect to the virtual machine that you just created, using SSH.
 
 To do so, go to the resource page of your virtual machine and find its public IP address. Then use SSH to log on to the machine, using the user name `ubuntu` and the private key file that you downloaded during the creation of the virtual machine. For example:
 
     ssh -i Downloads/anbox-cloud-appliance_key.pem ubuntu@192.0.2.15
 
-## Install the appliance
+## Finish the installation
+
+Perform the following steps to finish the appliance installation on the virtual machine. If you are not already familiar with how to perform these steps, see the [tutorial on installing the appliance](https://discourse.ubuntu.com/t/22681) for detailed instructions.
+
```

Comment:
hmm, we need to attach Ubuntu Pro token before installing the appliance on azure, don't we?

Reply:
> hmm, we need to attach Ubuntu Pro token before installing the appliance on azure, don't we?

It is part of the installation steps. We are redirecting the users to the tutorial if they don't already know/remember. In the tutorial, under "Install the appliance" section, there is a step for attaching Ubuntu Pro token.

Reply:
Yes, I see that. But I feel like it'd be better if we explicitly call that out here.
I don't have a strong opinion on this though.
let's keep as is if it looks like lengthy and redundant to you.